### DISCUSSION TITLE
Webfollower Java.lang.RuntimeException 403  minimumVersion":"4.12.0

### DISCUSSION THREAD

**Discussion by fludki** [NightscoutFoundation/xDrip#3809] [Bug reports]
Using the Webfollower and it keeps logging: 
Got exception 2: java.lang.RuntimeException: 403 Got failure
19:58 response code: 403
("data":("minimumVersion":"4.12.0"),"status":920)

Version: 6fcec23-2024.11.17
Code: 2124111721

I appreciate any ideas. What I found so far did not solve it.

**Comment by syska** [comment]
Got the same issue.

There are two other discussions about this issue with other names:

- https://github.com/NightscoutFoundation/xDrip/discussions/3794
- https://github.com/NightscoutFoundation/xDrip/discussions/3808
